packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/cancel/cancel.routing.js

```diff
@@ -0,0 +1,26 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'telecom.telephony.billingAccount.alias.details.portabilities.cancel',
+    {
+      url: '/cancel?portabilityId',
+      views: {
+        modal: {
+          component: 'portabilitiesCancel',
+        },
+      },
+      layout: 'modal',
+      translations: {
+        value: ['.'],
+        format: 'json',
+      },
```

Comment:
Redundant with https://github.com/ovh/manager/pull/7722/files#diff-4149967121b10e38f08959888e0c592a95fadc443b6ef88608de55089a12097cR22

Reply:
It's applied.